src/core/stdc/math.d

```diff
@@ -171,10 +171,27 @@ version( CRuntime_DigitalMars )
     }
   }
 }
-else version( CRuntime_Microsoft )
+else version( CRuntime_Microsoft ) // requires MSVCRT >= 2013
 {
     enum
     {
+        FP_SUBNORMAL = -2,
+        FP_NORMAL    = -1,
+        FP_ZERO      =  0,
+        FP_INFINITE  =  1,
+        FP_NAN       =  2,
+    }
+
+    private short _fdclass(float x);
+    private short _dclass(double x);
+
+    private int _fdsign(float x);
+    private int _dsign(double x);
+
+  deprecated("Please use the standard C99 functions instead.")
```

Comment:
I recommend emitting more specific deprecation messages here, but just for functions that have a backward compatible alternative. IMO just a comment would be ok, too.